src/index.js

```diff
@@ -292,7 +265,10 @@ class Dropzone extends React.Component {
       ...props // eslint-disable-line prefer-const
     } = rest;
 
-    const { isDragActive, isDragReject } = this.state;
+    const { draggedFiles } = this.state;
+    const isMultipleAllowed = multiple || draggedFiles.length <= 1;
+    const isDragActive = draggedFiles.length !== 0 && this.allFilesAccepted(draggedFiles);
```

Comment:
I'd prefer a comparison like `draggedFiles.length > 0` here and on the next line. I think it shows the intention better.